spec/predicates/cyclonedx.md

```diff
@@ -0,0 +1,71 @@
+# Predicate type: CycloneDX
+
+Type URI: (tentative) https://cyclonedx.org/bom
+
+Version: 1.0.0
+
+## Purpose
+
+A Software Bill of Materials type following the [CycloneDX standard].
+
+This allows representing "exportable", or "published" software artifacts,
+services, vulnerability information, and more. For a complete list of
+capabilities see [CycloneDX Capabilities].
+
+## Prerequisites
+The in-toto [attestation] framework.
+
+## Model
+This is a predicate type that fits within the larger [Attestation] framework.
+
+## Schema
+The schema of this predicate type is documented in the
+[CycloneDX Specification].
+
+### Parsing Rules
+The parsing rules for this predicate type are documented in the
+[CycloneDX Specification].
+
+### Fields
+The fields that make up this predicate type are documented in the
+[CycloneDX Specification].
+
+## Example
+```jsonc
+{
+  // Standard attestation fields:
+  "_type": "https://in-toto.io/Statement/v0.1",
+  "subject": [{ ... }],
+
+  // Predicate:
+  "predicateType": "https://cyclonedx.org/BOM",
```

Comment:
This should match https://cyclonedx.org/specification/overview/#recognized-predicate-type

```suggestion
  "predicateType": "https://cyclonedx.org/bom/v1.4",
```

Reply:
Is it supposed to _just_ be `https://cyclonedx.org/bom`?  The page doesn't include the version (which I assume is embedded in the `predicate` field as it is for SPDX?